fontir/src/glyph.rs

```diff
@@ -62,6 +62,7 @@ fn split_glyph(glyph_order: &GlyphOrder, original: &Glyph) -> Result<(Glyph, Gly
     simple_glyph.sources.iter_mut().for_each(|(_, inst)| {
         inst.components.clear();
     });
+    simple_glyph.codepoints.clear();
```

Comment:
See https://github.com/googlefonts/fontc/issues/457#issuecomment-2008651486

Reply:
thanks for spotting this! turns out not raising an error in case of cmap conflicts was actually hiding another bug :)